lang/en/campaigns.php

```diff
@@ -197,8 +197,7 @@
         'switch_back_success'   => 'Switched back to your account.',
         'title'                 => 'Members - :name',
         'updates'               => [
-            'added'     => 'Role :role added to :user.',
-            'removed'   => 'Role :role removed from :user.',
+            'roles'     => 'Campaign roles successfully updated for :user',
```

Comment:
Can just be "Roles" (I'm hoping to move away from the Campaign terminology later this year)